clang/lib/APINotes/APINotesReader.cpp

```diff
@@ -1822,7 +1836,10 @@ auto APINotesReader::lookupObjCClassID(StringRef name)
   if (!classID)
     return std::nullopt;
 
-  auto knownID = Impl.ObjCContextIDTable->find({*classID, '\0'});
+  // TODO: Can ObjC classes be declared in C++ namespaces? If so, we should pass
```

Comment:
No, Objective-C classes cannot be declared in C++ namespaces.

Reply:
Thanks, removed the comment